samples/javascript/groupchatgpt/Readme.md

```diff
@@ -21,8 +22,12 @@ In some of the other samples, we show how to run the app locally and expose the
 
 In this sample, we open this project from [GitHub codespace](https://github.com/features/codespaces) and run it from inside the Codespace.
 
-1. In GitHub repo, click Code and choose `Codespaces` tab to open the project in codespace.
-2. In Codespace, switch to the Terminal tab:
+1. In this GitHub repo, click **Code** and choose `Codespaces` tab to open the project in codespace. If there is no codespace yet, click *Create codespace on main* to create one for you. Codespace starts in seconds with up to 60 hours a month free.
+2. In Codespace, switch to the Terminal tab
+    0. Navigate to current folder
```

Comment:
Looks the `0.` leads to some rendering issue.

![image](https://user-images.githubusercontent.com/15338714/231976074-f563e590-2aa2-4a55-8e01-5c1a81832f61.png)